sigilrecorder.lic

```diff
@@ -0,0 +1,297 @@
+=begin
+  Documentation: https://elanthipedia.play.net/Lich_script_repository#sigilrecorder
+=end
+
+custom_require.call(%w[common common-arcana common-items common-travel drinfomon events spellmonitor equipmanager])
+
+class SigilRecorder
+	require 'yaml'
+
+	def initialize
+    arg_definitions = [
+      [
+        { name: 'debug', regex: /debug/i, optional: true, description: 'Show debug messaging' }
+      ]
+    ]
+
+    args = parse_args(arg_definitions)
+
+    args.debug == 'debug' ? @debug = true : @debug = false
```

Comment:
🧹 I believe this can be simplified to this assignment.
```suggestion
    @debug = args.debug
```
If `parse_args` sees `debug` as an argument then `args.debug` will have the value "debug", which in Ruby is truthy because it's neither nil nor false.

If `parse_args` does not see `debug` as an argument then `args.debug` will be nil, which is falsey.

Reply:
That melts my brain a bit but it also sorta makes sense haha. I'll change it over and test it out!